addons/mass_mailing/views/themes_templates.xml

```diff
@@ -2,10 +2,10 @@
 <odoo>
     <!-- Theme "Basic" template -->
     <template id="theme_basic_template">
-        <div class="o_mail_no_options">
+        <div class="o_mail_no_options" style="font-size: 14px;">
```

Comment:
This is not the correct way to change the font-size it would make more sense to modify the variable `--body-font-size` to 1rem when we have a body that contains the basic_theme.
This rule could be done in `theme_basic.scss`.